reviews three/swagger.yaml

```diff
@@ -0,0 +1,954 @@
+              }
+              
+              begin
+                result = api_instance.admin_draft_orders_json_get(opts)
+                p result
+              rescue SwaggerClient::ApiError => e
+                puts "Exception when calling GetApi->admin_draft_orders_json_get: #{e}"
+              end
+            
+            
+  
+    post:
+      summary: Create a draft order
+      description: |
+          
+          Using the Draft Orders API you can create draft orders using product variant line items, or your own custom line items.  This enables you to create one-off products that exist outside of your regular inventory management system.  To create a product variant draft order, simply specify a `variant_id` `quantity` and `discount` (if applicable).  Create a custom line item by supplying `title`, `price`, `taxable` and `quantity`. 
+          
+        
+          | `Note`| The array of line items on a draft order must not contain a `variant_id` more than once. The data copied over from a variant upon creation of a line item is never subsequently updated.  |
+          |---|-------------|
+          
+          #### About Custom Shipping Lines
+          
+            Using the Draft Orders API provides you with the flexibility to fulfill order shipments either using Shopify or outside of Shopify.  To fulfill your order within Shopify, you can send the `handle` of the `shipping_rate` set via the Shopify admin.  For a custom shipping line, simply specify a `shipping_line` object with no `handle`.  This is covered in more detail in the endpoint calls below.  
+         
+          
+      tags:
+        - Create
+
+      parameters:
+        - name: draft_order
+          in: body
+          description: draft_order to create.
+          schema:
+            allOf:
+              - $ref: '#/definitions/DraftOrder'
+            required:
+            - line_items
+      
+          
+            
+
+      responses:
+        '200':
+          description: A single DraftOrder
+          schema:
+            $ref: '#/definitions/DraftOrder'
+            
+      x-code-samples:
+        - lang: curl
+          source: |
+            curl -X POST -H "Content-Type: application/json" -H "Authorization: Basic YOUR-API-KEY=" -d '{
+              "id": 57760135,
+              "order_id": 57760134,
+              "name": "api created draft order",
+              "customer": {
+                "accepts_marketing": true,
+                "created_at": "2016-11-18T17:10:07Z",
+                "email": "string",
+                "first_name": "string",
+                "order_id": 0,
+                "last_name": "string",
+                "note": "string",
+                "orders_count": 0,
+                "state": "string",
+                "total_spent": 0,
+                "updated_at": "2016-11-18T17:10:07Z",
+                "tags": "string"
+              },
+              "shipping_address": "string",
```

Comment:
This has multiple fields, not just a string. 
Billing address too. See the checkout API docs for examples.